source/braille.py

```diff
@@ -380,6 +387,52 @@
 BLUETOOTH_PORT = ("bluetooth", _("Bluetooth"))
 
 
+class formattingSymbol(NamedTuple):
```

Comment:
syntax for classes is UpperCamelCase

```suggestion
class FormattingSymbol(NamedTuple):
```